modules/rosa-accessing-your-cluster.adoc

```diff
@@ -4,9 +4,14 @@
 
 
 [id="rosa-accessing-your-cluster"]
```

Comment:
Please refer to my previous comment on IDs in modules.

Reply:
Done,